src/modules/CC1101/CC1101.cpp

```diff
@@ -517,9 +521,31 @@ int16_t CC1101::setFrequencyDeviation(float freqDev) {
   // set frequency deviation value
   int16_t state = SPIsetRegValue(RADIOLIB_CC1101_REG_DEVIATN, (e << 4), 6, 4);
   state |= SPIsetRegValue(RADIOLIB_CC1101_REG_DEVIATN, m, 2, 0);
+  
   return(state);
 }
 
+int16_t CC1101::getFrequencyDeviation(float *freqDev) {
```

Comment:
All pointers should be checked if not NULL - this applies to all getters as well.

Reply:
I added a proper `RADIOLIB_ERR_NULL_POINTER` error code to account for this.